src/Ocelot/Configuration/Builder/MetadataOptionsBuilder.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Globalization;
+
+namespace Ocelot.Configuration.Builder;
+
+public class MetadataOptionsBuilder
```

Comment:
Why is there no interface available?
Could it be relocated to the Metadata folder?

Reply:
@raman-m what the heck are you talking about???